var/spack/repos/builtin/packages/py-beancount-fava/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBeancountFava(PythonPackage):
```

Comment:
According to PyPI, the `beancount-fava` package has been renamed to `fava`

Reply:
Ok, I've renamed it now.